charm/zookeeper/files/check_zookeeper.py

```diff
@@ -174,9 +176,23 @@ def analyze(self, opts, cluster_stats):
 
 class ZooKeeperServer(object):
 
-    def __init__(self, host='localhost', port='2181', timeout=1):
+    def __init__(self, host='localhost', port='2181', timeout=1,
+                 meta_file='/tmp/zk_check/meta'):
         self._address = (host, int(port))
         self._timeout = timeout
+        self._last_reset = datetime.utcnow()
+        self._meta_path = meta_file
+
+        directory = os.path.dirname(meta_file)
```

Comment:
These 3 lines can be replaced with `os.makedirs(os.path.dirname(meta_file), exist_ok=True)`